scripts/run_pyuvsim.py

```diff
@@ -7,15 +7,13 @@
 
```

Comment:
This script should also be deprecated. The only supported wrapper scripts are `run_param_pyuvsim.py` and `run_profile_pyuvsim.py`. This script will break the MPI functions.

Reply:
Removed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0